packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/fields.tsx

```diff
@@ -194,9 +197,11 @@ const QuickStrategyFields = React.memo(
                                         selected_symbol={selected_symbol}
                                     />
                                 </div>
-                                {id.endsWith('types-strategies') && (
-                                    <div key='description' className='quick-strategy__description'>
-                                        {description}
+                                {id.endsWith('type-strategy') && (
```

Comment:
actually I don’t think it’s a good idea to string comparing for finding the category or type
It’s better to add the category as a prop (object enumeration)… then whenever you wanna add a new entry, you can simply add in the object … 
string comparing make the maintenance harder

Reply:
@sara-fs actually after more changes we don’t need any more check the string like this `id.endsWith('type-strategy') `. We have `id`
```
    {
        id: 'type-strategy',
        field_name: 'quick-strategy__type-strategy',
        className: '',
        select_value: 'type-strategy',
        label: '',
        is_basic_field: true,
    },
    {
        id: 'symbol',
        field_name: 'quick-strategy__symbol',
        className: 'quick-strategy__dropdown quick-strategy__leading',
        select_value: 'symbol',
        label: 'Asset',
        is_basic_field: true,
    },
```
What do you mean here "It’s better to add the category as a prop (object enumeration)… then whenever you wanna add a new entry, you can simply add in the object …" ??

Reply:
we can use `id === 'type-strategy'` instead of `id.endsWith('type-strategy')`. We need to know that this is a select type strategy and we wonna add something after it

Reply:
what is wrong in this approach ?

Reply:
I mean if we have different types, for example [strategy, .... ] we can have an object like:
    const type = Object.freeze({
        0: 'strategy',
        1: '.....',
        2: '...',
    });

and then compare the id with these enums. It's a better practice instead of comparing strings together ...

Reply:
@sara-fs  In the current case, we are inside of the `.map()`. In this case for additional, we need to add logic what is the object related to current `0: 'strategy',` for example. we need to spread and pass properties regarding this special type(Object.freeze({...})). What difference does it make then?